Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -151,6 +154,50 @@ void AODProducerWorkflowDPL::createCTPReadout(const o2::globaltracking::RecoCont
     }
   }
   LOG(info) << "# of TRD bogus triggers:" << cntwarnings;
+
+  // EMCAL:
+  cntwarnings = 0;
+  orbitPrev = 0;
+  bcPrev = 0;
+  // Get class mask for EMCAL:
+  uint64_t classMaskEMCAL = 0;
+  for (const auto& trgclass : ctpcfg->getCTPClasses()) {
+    if (trgclass.cluster->getClusterDetNames().find("EMC")) {
+      classMaskEMCAL = trgclass.classMask;
```

Comment:
should not one break here?

Reply:
Ideally I would like to add PHOS and TRD into the same loop, then I can't break but need to check all cases separately, then it could be passes as argument to a lambda function which is generic for all detectors. As I wrote previously I see some issue with the CCDB objects which might lead to incorrect trigger classes in the TRD case. However I first would like to test it with EMCAL alone. While doing the test the PR I will label the PR as [WIP].